op-node/p2p/prepared.go

```
@@ -92,3 +95,10 @@ func (p *Prepared) Disabled() bool {
 func (p *Prepared) ReqRespSyncEnabled() bool {
 	return p.EnableReqRespSync
 }
+
+func (p *Prepared) GetGossipTimestampThreshold() time.Duration {
+	if p.GossipTimestampThreshold == 0 {
+		return 60 * time.Second // Default fallback
+	}
+	return p.GossipTimestampThreshold
+}
```

Is this getter necessary? We have the default value at the CLI level. We could just `panic` if we try to start `op-node` with an unset value?

---

The getter is necessary because I added it to the `Config` interface. However the explicit check for 0 value with a default is not necessary so I removed it.